Resources/config/mobile_detect.xml

```diff
@@ -17,6 +17,7 @@
         <service id="mobile_detect.mobile_detector.default" class="%mobile_detect.mobile_detector.class%"/>
 
         <service id="mobile_detect.device_view" class="%mobile_detect.device_view.class%">
+            <argument>%mobile_detect.redirect%</argument>
```

Comment:
@suncat2000 i made the refactoring but travis is still running after more 1 hour.